src/test/java/io/stargate/sgv2/jsonapi/api/v1/InAndNinIntegrationTest.java

```diff
@@ -576,18 +639,46 @@ public void nonIdNinEmptyArray() {
           .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
           .then()
           .statusCode(200)
-          .body("data.documents", hasSize(6))
+          // except doc 5
+          .body("data.documents", hasSize(5))
           .body("status", is(nullValue()))
-          .body("errors", is(nullValue()));
+          .body("errors", is(nullValue()))
+          .body("data.documents", notNullValue());
     }
```

Comment:
Why this test doesn't have the expected value? Perhaps you could add `.body("data.documents", hasSize(5))` if you don't want to list them all.

Reply:
thanks fixed.